src/warc2zim/statics/wombat_setup.js

```diff
@@ -0,0 +1,197 @@
+// This file setup a wbinfo dictionnary to configure a wombat instance.
+// It mainly export one function `getWombatInfo` which takes context information
+// and returns a configuration dictionnary to pass to wombat.
+
+const getWombatInfo = (function () {
+  // Reduce "complex" url to our simplyfied ones.
+  // ! MUST be in sync with the python code (at zim creation time)
+  let reduce = function (path) {
+    const fuzzy_rules = [
+      {
+        match: new RegExp(
+          ".*googlevideo.com/(videoplayback\\?).*((?<=[?&])id=[^&]+).*",
+        ),
+        replace: "youtube.fuzzy.replayweb.page/$1$2",
+      },
+      {
+        match: new RegExp(
+          "(?:www\\.)?youtube(?:-nocookie)?\\.com/(get_video_info\\?).*(video_id=[^&]+).*",
+        ),
+        replace: "youtube.fuzzy.replayweb.page/$1$2",
+      },
+      {
+        match: new RegExp("(\\.[^?]+\\?)[\\d]+$"),
+        replace: "$1",
+      },
+      {
+        match: new RegExp(
+          "(?:www\\.)?youtube(?:-nocookie)?\\.com\\/(youtubei/[^?]+).*(videoId[^&]+).*",
+        ),
+        replace: "youtube.fuzzy.replayweb.page/$1?$2",
+      },
+      {
+        match: new RegExp(
+          "(?:www\\.)?youtube(?:-nocookie)?\\.com/embed/([^?]+).*",
+        ),
+        replace: "youtube.fuzzy.replayweb.page/embed/$1",
+      },
+      {
+        match: new RegExp("youtube.fuzzy.replayweb.page/embed/([^?&]+).*"),
+        replace: "youtube.fuzzy.replayweb.page/embed/$1",
+      },
+      {
+        match: new RegExp(
+          ".*(?:gcs-vimeo|vod|vod-progressive)\\.akamaized\\.net.*?/([\\d/]+.mp4)$",
+        ),
+        replace: "vimeo-cdn.fuzzy.replayweb.page/$1",
+      },
+      {
+        match: new RegExp(".*player.vimeo.com/(video/[\\d]+)\\?.*"),
+        replace: "vimeo.fuzzy.replayweb.page/$1",
+      },
+    ];
+
+    for (const rule of fuzzy_rules) {
+      const new_path = path.replace(rule.match, rule.replace);
+      if (new_path != path) {
+        return new_path;
+      }
+    }
+    return path;
+  };
+
+  const getWombatInfo = function (
```

Comment:
Having a variable named `getWombatInfo` inside another variable named `getWombatInfo` does not really help in readability, it took me a while to understand this relation.

Reply:
This is now replaced by a "direct" return:

```js
return function(
...
```